include/laser_assembler/base_assembler.h

```diff
@@ -56,7 +58,7 @@ namespace laser_assembler
 /**
  * \brief Maintains a history of point clouds and generates an aggregate point cloud upon request
  */
-template<class T>
+template<class T, class V>
```

Comment:
I don't understand in which way this should be done. Wouldn't it be to much overhead? Maybe a good way is to create a deprecated class: base_assembler with V=sensor_msgs::PointCloud What do you think?